README.md

```diff
@@ -58,7 +58,7 @@ You can include it in a `requirements.yml` file and install it via `ansible-gala
 ---
 collections:
   - name: community.postgresql
-    version: 0.1.0
+    version: 1.0.0
```

Comment:
I think it's better to leave `version:` off, otherwise people will never get updates because they won't update the version.

Reply:
good suggestion! I copy-pasted this from community.mysql. So, i'll remove this from there as well, thanks!